diff --git a/node_modules/react-native-signature-capture/ios/PPSSignatureView.m b/node_modules/react-native-signature-capture/ios/PPSSignatureView.m
index ca2801b..5c48736 100644
--- a/node_modules/react-native-signature-capture/ios/PPSSignatureView.m
+++ b/node_modules/react-native-signature-capture/ios/PPSSignatureView.m
@@ -150,10 +150,6 @@ - (void)commonInit {
 		tap.cancelsTouchesInView = YES;
 		[self addGestureRecognizer:tap];
 		
-		// Erase with long press
-		UILongPressGestureRecognizer *longer = [[UILongPressGestureRecognizer alloc] initWithTarget:self action:@selector(longPress:)];
-		longer.cancelsTouchesInView = YES;
-		[self addGestureRecognizer:longer];
 	}
 	else
 		[NSException raise:@"NSOpenGLES2ContextException" format:@"Failed to create OpenGL ES2 context"];
@@ -378,11 +374,6 @@ - (void)tap:(UITapGestureRecognizer *)t {
 	[self setNeedsDisplay];
 }
 
-
-- (void)longPress:(UILongPressGestureRecognizer *)lp {
-	[self erase];
-}
-
 - (void)pan:(UIPanGestureRecognizer *)p {
 	
 	glBindBuffer(GL_ARRAY_BUFFER, vertexBuffer);
